polkadot/node/network/collator-protocol/src/collator_side/mod.rs

```diff
@@ -200,7 +206,7 @@ struct PeerData {
 }
 
 /// A type wrapping a collation and it's designated core index.
-struct CollationWithCoreIndex(Collation, CoreIndex);
+struct CollationWithCoreIndex(Collation, CoreIndex, Option<CollationStats>);
```

Comment:
This should likely have a different name now and would be better suited to a regular struct (not a tuple struct)